service/migration_manager.cc

```diff
@@ -132,6 +133,9 @@ void migration_manager::init_messaging_service()
                     "migration request handler: group0 snapshot transfer requested, but canonical mutations not supported");
             }
             cm.emplace_back(co_await db::system_keyspace::get_group0_history(proxy));
+            for (auto&& m : co_await replica::read_tablet_mutations(proxy)) {
+                cm.emplace_back(std::move(m));
+            }
```

Comment:
Is this viable? A database with 100TB storage and 10GB tablets will have 10k tablets.

I guess it's okay for now, but we'll have to replace it with a stream later on.

Reply:
Yes, we need to switch to streaming at some point. 

perf_tablets shows that 10k is fine, but with 100k we hit the commitlog mutation size limit.

Reply:
ok

Reply:
btw we'll hit it sooner, e.g. 20TB per datacenter * 5 datacenters, so not a huge database.